vars/fhcapNode.groovy

```diff
@@ -25,58 +25,54 @@ def call(Map parameters = [:], body) {
 
     node(withLabels(labels)) {
 
-        def utils = new Utils()
-        def installLatest = parameters.get('installLatest', false)
-        def credentialsId = parameters.get('credentialsId', 'jenkinsgithub')
-        def configFile = parameters.get('configFile', "${WORKSPACE}/fhcap.json")
-        def gitRepo = parameters.get('gitRepo', null)
-        def gitRef = parameters.get('gitRef', 'master')
-        def gitUserName = parameters.get('gitUserName', null)
-        def gitUserEmail = parameters.get('gitUserEmail', null)
-        def fhcapRepos = parameters.get('fhcapRepos', null)
-        def workDir = parameters.get('workDir', '.')
-        def setupOpts = parameters.get('setupOpts', [:])
-        def defaultSetupOts = ['--repos-dir': WORKSPACE, '--fh-src-dir': WORKSPACE, '--knife-dir': WORKSPACE]
-
-        String setupOptsStr = utils.mapToOptionsString(defaultSetupOts + setupOpts, ' ')
-
-        step([$class: 'WsCleanup'])
-        sshagent([credentialsId]) {
-
-            sh "rm -rf ${HOME}/fhcap"
-
-            if (gitUserName && gitUserEmail) {
-                sh "git config --global user.name \"${gitUserName}\""
-                sh "git config --global user.email \"${gitUserEmail}\""
-            }
-
-            ansiColor('xterm') {
-                env.RUBYOPT = "-W0"
-
-                if (gitRepo) {
-                    checkoutFhcapCli(gitRepo, gitRef, true)
-                } else {
-                    //This is only here to get git clones to work correctly, and avoid 'Host key verification failed' errors
-                    checkoutFhcapCli()
-                    if (installLatest) {
-                        sh "gem install fhcap-cli --no-ri --no-rdoc"
-                    }
+        def configFile = parameters.get('configFile', "${env.WORKSPACE}/fhcap.json")
+        withEnv(["PATH+=/home/jenkins/bin", "RUBYOPT=-W0", "FHCAP_CFG_FILE=${configFile}"]) {
```

Comment:
does `"PATH+=/home/jenkins/bin"` work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be `"PATH+=:/home/jenkins/bin"`, does that sound right?

Reply:
> does "PATH+=/home/jenkins/bin" work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be "PATH+=:/home/jenkins/bin", does that sound right?

Nope :)

Reply:
> does "PATH+=/home/jenkins/bin" work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be "PATH+=:/home/jenkins/bin", does that sound right?

Nope :)
```
"PATH+=:/home/jenkins/bin"
```
:/home/jenkins/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/var/lib/jenkins/.local/bin

Reply:
Ah right, it's different behaviour in a pipeline than in a shell -- the shell also adds the new part as a suffix, where the pipeline adds it as a prefix.